README.md

```diff
@@ -441,12 +441,12 @@ The registration of the service account is handled automatically during ``make
 deploy``. To register  without deploying, run ``make -C terraform sam``. The
 authorization must be performed manually by a developer with administrator access to
 the SAM group that controls access to TDR. Currently that group is called
-`JadeStewards-dev`. *Jade* is the Broad-internal codename for TDR.
+`azul-dev`.
 
-A member of the *Jade stewards* group has read and write access to TDR, and an
+A member of the *azul-dev* group has read and write access to TDR, and an
```

Comment:
A distinction needs to be made between English prose and technical strings. `JadeStewards-dev` is computer-speak, but "Jade Stewards" is an English description (the stewards of Jade). The former should always have `code-formatting` and the latter shouldn't.

`azul-dev` is a technical string and therefore should have code formatting, like so.